scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -278,7 +278,7 @@ class FossId internal constructor(
                 }
 
             if (filteredPackages.isEmpty()) {
-                log.warn { "There is no package to scan !" }
+                log.warn { "There is no package to scan." }
```

Comment:
Consider making this `.info`.

Reply:
I prefer to keep this a warning as this case should be rather unusual (there should be at least the project package).